src/components/Navigation.tsx

```diff
@@ -25,7 +25,7 @@ export interface NavigationSection {
   path: string;
   role?: string;
   badge?: { count: number };
-  url?: string;
+  url: string;
```

Comment:
Question: Was this just a temporary change from the last PR in #1334 that's reverted now that a URL is always passed or more to it?

Reply:
Correct. The URL is a required parameter to actually jump between the different sections.